charts/prometheus-postgres-exporter/templates/deployment.yaml

```
@@ -52,10 +52,10 @@ spec:
           - "--extend.query-path=/etc/config.yaml"
           - "--web.listen-address=:{{ .Values.service.targetPort }}"
           {{- if .Values.config.disableCollectorDatabase }}
-          - "--no-collector.database"
+          - "--no-collector.stat_database"
```

```suggestion
          - "--no-collector.database"
```

Thank you, @mjasion. The [database collector](https://github.com/prometheus-community/postgres_exporter/blob/master/collector/pg_database.go) and the [stat_database collector](https://github.com/prometheus-community/postgres_exporter/blob/master/collector/pg_stat_database.go) are two different collectors, i.e. this is not a case of a changed flag. The chart currently supports disabling the former but not the latter.

---

Ah, didnt know. Thx for catching this.